CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `Popup` styling with `pointer` when it is wrapped by `FocusZones` @layershifter ([#1492](https://github.com/stardust-ui/react/pull/1492))
 - Apply custom focus style on `TreeItem` and `TreeTitle` @silviuavram ([#1506](https://github.com/stardust-ui/react/pull/1506))
 - Fix `theme` types, remove duplication @kuzhelov ([#1508](https://github.com/stardust-ui/react/pull/1508))
+- Fix `Dropdown` and `DropdownItem` styles to match redlines ([#1523] (https://github.com/stardust-ui/react/pull/1523))
```

Comment:
For style related changes, the request is to have the comment include that it's for Teams theme and an indicator for consumers to know they might need to remove style overrides with a `[redlines]` identifier. So something like

```
Fix `Dropdown` and `DropdownItem` Teams theme styles to match redlines [redlines]
```